native/membrane_videocompositor_opengl_rust/build.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+  println!("cargo:rustc-link-lib=EGL");
+  println!("cargo:rustc-link-lib=GLESv2");
+  println!("cargo:rustc-link-search=../..");
```

Comment:
MacOS doesn't have EGL, which we need to use for this project. We use [Google ANGLE](https://github.com/google/angle) on MacOS, and we ended up just putting the two libraries in the project root if we're compiling on MacOS. That was back when we were using C++ for the prototype and we had to do it because of Unifex weirdness on MacOS. When we started using Rust, the libraries were already established to be there.